src/services/getUserService.js

```diff
@@ -0,0 +1,21 @@
+const { queryByUsername } = require("../repository/getUserDAO");
+
+
+async function getUserByUsername(req, res) {
+    const username = req.body.username;
+    const result = await queryByUsername(username);
+    const statusCode = result?.$metadata?.httpStatusCode;
+    const foundUser = result?.Item;
+    if (statusCode == 200) {
+        res.status(statusCode).json({
```

Comment:
I wouldn't send the response status code of us interacting with the database back to the user. We'd lose a lot of control on what we send back on errors. For example it could be user input error which would be a 400 or it could be that dynamodb is down which would be a 500.